include/flmpi.h

```diff
@@ -0,0 +1,45 @@
+/* Copyright (C) 2006 Imperial College London and others.
+
+ Please see the AUTHORS file in the main source directory for a full list
+ of copyright holders.
+
+ Dr Gerard J Gorman
+ Applied Modelling and Computation Group
+ Department of Earth Science and Engineering
+ Imperial College London
+
+ g.gorman@imperial.ac.uk
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2.1 of the License.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library; if not, write to the Free Software
+ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307
+ USA
+*/
+#ifndef FLMPI_H
+#define FLMPI_H
+
+#include "confdefs.h"
+#ifdef HAVE_MPI
+#if !defined(MPICH_SKIP_MPICXX)
+#  define MPICH_SKIP_MPICXX 1
+#endif
+#if !defined(OMPI_SKIP_MPICXX)
+#  define OMPI_SKIP_MPICXX 1
+#endif
+#if !defined(OMPI_WANT_MPI_INTERFACE_WARNING)
+#  define OMPI_WANT_MPI_INTERFACE_WARNING 0
+#endif
```

Comment:
What's this one for?

Reply:
Good point I just copied that randomly from petsc. It seems to supresswarnings about deprecated mpi functions, which I guess we would want to know about.